### DISCUSSION TITLE
Some ideas for improvment

### DISCUSSION THREAD

**Discussion by duje13** [trakt/showly#485] [Ideas / Feature Requests]
Hi,

I'm using this app for several months now, I this is really love it (I rate 5 on google play) but here are some improvements and features I found can be done. I'm sorry if this is not best place for this, but it seems to me more appropriate then issues and since I'm also programmer I found it better here then google play review. Also sorry for my bad English :)

**POSSIBLE BUG WITH SYNC**

This is probably problem with my mobile (One Plus 3T btw) not the app, but it worth to mention if you found some time to look at it. Anyway, I set it up to sync every day but it someday sync, some not, pretty randomly... I know some smartphones have problem with aggressively killing background tasks, and I do all steps I found on internet to prevent it, but this is probably still related to that.

**IMPROVEMENTS**

1. Search bar is little confusing. It search only current tab (so to search all movies I need to go to discover tab). In my opinion it is not good metal model, from other apps we are used to that big search bar on top of the screen is always main search of everything. All the times I found myself trying to search some movie and then figuring out "oh i'm on wrong tab that why there is no movie!" So my suggestion is to search bar on top of screen is always do what search bar in discover tab currently do and to put search of lists somewhere else. 
2. Another related to search bar. This one is not so big problem, only first time until you figure out how to use it. When I search for some movie it give me just few movies and to get all results you need to hit enter. For some who use it first time it is not so obvious. I definitely need few minutes to figure it out :) But I as I say it not big problem and maybe it just up to me.

**NEW FEATURES**

1. Rate by season - this would be grate thing, to not be able only to rate complete TV show or by single episode but also by season. Sometimes difference between seasons are really big and rate every single episode is a bit overhead (at least for me). Unfortunately I didn't see it on any similar app. It would be cool if you somehow made it, but I know you are probably limited by API (I'm not sure does trakt.tv support it?)
2. Possibility to mark TV show as not watching currently - this would be really important and useful feature to me. So idea it that I can mark TV show not watching currently which will make it disappear from progress bar (but not calendar I still want to now when new season or episode comes out). Sometimes I just don't watch some show currently (I often wait to whole season come out, for example) and during that time i don't want it to mess my progress tab.
3. Episode duration - show episode duration when I click on it. It would be useful information and trakt.tv has it, so I supposed it is not complicated to do.  
4. Remember when movie/TV show is watched - when I mark on something that I watched it i would be nice to remember that date so later on I can recall when I watch it.
5. Personal notes - it would be nice so I can add some notes about TV show. It not same as comments, comments are for public (review) which purpose to give advice to other people why to watch it or not. This would be my personal notes (for example who I watched TV show with, or why I stopped watched it...)

**Comment by blackholeearth** [comment]
About new features -> no2

İt could be good. İf app pushed them below to new section calling  "still airing" . 
So we can wait until all episodes of new season are released . 
dont think about it .
Focus on already tv series that released full seasons
